content/en/docs/concepts/storage/volumes.md

```diff
@@ -1158,7 +1158,7 @@ spec:
 
 
 Use the `subPathExpr` field to construct `subPath` directory names from Downward API environment variables.
-Before you use this feature, you must enable the `VolumeSubpathEnvExpansion` feature gate.
+This feature is enabled by default.
```

Comment:
Readers might still want to know the name of the feature gate that controls subPath expansion.

Reply:
@houjun41544, do you agree with my comment? I recommend telling readers the name of the feature gate, and also adding a link to the page about feature gates. (That's how most similar pages handle it).

Reply:
@sftim Sorry, I was out on vacation these days so I  didn't see your comment.
I agree with your suggestion.